content/en/tracing/code_origins/_index.md

```diff
@@ -89,13 +103,20 @@ export DD_CODE_ORIGIN_FOR_SPANS_ENABLED=true
 
 ## Troubleshooting
 
-If Code Origin information is missing:
+### Code Origin section is missing ###
+
+- Verify Code Origins is [enabled](#enable-code-origins) in your tracing library configuration.
+- Confirm that your service meets all [compatibility requirements](#compatibility-requirements) (ie. service language, supported frameworks, and minimum tracer version).
+- For most services, Code Origins data will be captured for [service entry spans][12] only. You can filter to "Service Entry Spans" in the [APM Trace Explorer][1].
+
+    {{< img src="tracing/code_origins/code_origin_service_entry_spans_filter.png" alt="Code Origins - Search for Service Entry Spans" style="width:100%;">}}
+
+- To search for all spans that include Code Origins, use the query `@_dd.code_origin.type:*` in the [APM Trace Explorer][1].
+
+### Code preview is not visible ###
+- Ensure that all [Source Code Integration][7] setup requirements are met.
+- For transpiled Node.js applications (eg. Typescript), make sure to generate and publish source maps with the deployed application, and to run Node.js with the [`--enable-source-maps`][10] flag. Otherwise, code previews will not work. See the Node.js [Source Code Integration][9] documentation for more details. 
```

Comment:
```suggestion
- For transpiled Node.js applications (for example, Typescript), make sure to generate and publish source maps with the deployed application, and to run Node.js with the [`--enable-source-maps`][10] flag. Otherwise, code previews do not work. See the Node.js [Source Code Integration][9] documentation for more details. 
```